src/core/Akka.Discovery/Config/ConfigServiceDiscovery.cs

```diff
@@ -0,0 +1,60 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="Lease.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2020 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2020 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Annotations;
+using Akka.Event;
+
+namespace Akka.Discovery.Config
+{
+    [InternalApi]
+    public class ConfigServicesParser
+    {
+        public static Dictionary<string, ServiceDiscovery.Resolved> Parse(Configuration.Config config)
+        {
+            return config.AsEnumerable()
+                .Select(pair => (pair.Key, config.GetConfig(pair.Key)))
+                .ToDictionary(pair => pair.Key, pair =>
+                {
+                    var (serviceName, full) = pair;
+                    var endpoints = full.GetStringList("endpoints");
```

Comment:
Missing `GetConfigList`, not currently supported

Reply:
@Arkatufus is this something we need to add to the HOCON spec?

Reply:
I've linked a issue on the HOCON repo to this